types/google-map-react/index.d.ts

```diff
@@ -66,30 +66,191 @@ declare namespace googleMapReact {
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            INVALID_REQUEST: string;
+            ZERO_RESULTS: string;
+            ERROR: string;
+        };
+        KmlLayerStatus: {
+            UNKNOWN: string;
+            OK: string;
+            INVALID_REQUEST: string;
+            DOCUMENT_NOT_FOUND: string;
+            FETCH_ERROR: string;
+            INVALID_DOCUMENT: string;
+            DOCUMENT_TOO_LARGE: string;
+            LIMITS_EXCEEDED: string;
+            TIMED_OUT: string;
+        };
+        MapTypeControlStyle: {
+            DEFAULT: number;
+            HORIZONTAL_BAR: number;
+            DROPDOWN_MENU: number;
+            INSET: number;
+            INSET_LARGE: number;
+        };
+        MapTypeId: {
+            ROADMAP: string;
+            SATELLITE: string;
+            HYBRID: string;
+            TERRAIN: string;
+        };
+        MaxZoomStatus: {
+            OK: string;
+            ERROR: string;
+        };
+        NavigationControlStyle: {
+            DEFAULT: number;
+            SMALL: number;
+            ANDROID: number;
+            ZOOM_PAN: number;
+            Vz: number;
+            Ot: number;
+        };
+        RenderingType: {
+            UNINITIALIZED: string;
+            RASTER: string;
+            VECTOR: string;
+        };
+        ScaleControlStyle: {
+            DEFAULT: number;
+        };
+        StreetViewPreference: {
+            NEAREST: string;
+            BEST: string;
+        };
+        StreetViewStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            ZERO_RESULTS: string;
+        };
+        StreetViewSource: {
+            DEFAULT: string;
+            OUTDOOR: string;
+        };
+        StrokePosition: {
+            CENTER: number;
+            INSIDE: number;
+            OUTSIDE: number;
+        };
+        SymbolPath: {
+            CIRCLE: number;
+            FORWARD_CLOSED_ARROW: number;
+            FORWARD_OPEN_ARROW: number;
+            BACKWARD_CLOSED_ARROW: number;
+            BACKWARD_OPEN_ARROW: number;
+        };
+        TrafficModel: {
+            BEST_GUESS: string;
+            OPTIMISTIC: string;
+            PESSIMISTIC: string;
+        };
+        TransitMode: {
+            BUS: string;
+            RAIL: string;
+            SUBWAY: string;
+            TRAIN: string;
+            TRAM: string;
+        };
+        TransitRoutePreference: {
+            LESS_WALKING: string;
+            FEWER_TRANSFERS: string;
+        };
+        TravelMode: {
+            DRIVING: string;
+            WALKING: string;
+            BICYCLING: string;
+            TRANSIT: string;
+            TWO_WHEELER: string;
+        };
+        UnitSystem: {
+            METRIC: number;
+            IMPERIAL: number;
+        };
+        ZoomControlStyle: {
+            DEFAULT: number;
+            SMALL: number;
+            LARGE: number;
+            Ot: number;
+        };
```

Comment:
Could not find `ZoomControlStyle` in [the documentation](https://developers.google.com/maps/documentation/javascript/reference). Where does it come from ?

Reply:
Please see [main comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60329#issuecomment-1126798899) for response on where it comes from. 

This appears to be depreacated and I will mark it as such [source](https://developers.google.com/maps/documentation/javascript/releases?hl=en#15-september-2015)